app/i18n/en/index.ts

```diff
@@ -642,6 +642,8 @@ const en: BaseTranslation = {
     IUnderstand: "I understand. Please log me out.",
     logoutAlertTitle: "Are you sure you want to log out and delete all local data?",
     logoutAlertContent: "You will need to re-enter your phone number to log back in.\nyour phone number is {phoneNumber: string} so make sure to have access to it to log back in",
+    usdBalanceWarning: "You have a balance of {balance: string} USD.",
+    btcBalanceWarning: "You have a balance of {balance: string} sats.",
```

Comment:
The balance string from `formatMoneyAmount` will include units. I would change these to be "You have a bitcoin balance of {balance: string}." and "You have a Stablesats balance of {balance: string}."